lib/graphql/utils.ex

```diff
@@ -71,6 +71,8 @@ defmodule BridgeEx.Graphql.Utils do
 
   @spec normalize_inner_fields(%{atom() => any()} | String.t()) :: %{atom() => any()} | String.t()
```

Comment:
This would need to be updated too

Reply:
Considering the other comment, I don't know if it's a good idea to specify all typespec like this
```suggestion
  @spec normalize_inner_fields(%{atom() => any()} | String.t() | boolean() | integer() | float()) :: %{atom() => any()} | String.t() | String.t() | boolean() | integer() | float()
```
risking not covering all cases.

Maybe in tihs case we can stay generic with 
```suggestion
  @spec normalize_inner_fields(any()) :: any()
```

Reply:
Yeah it makes sense! Could remove the `@spec` altogether if it's just `any -> any`?

Reply:
Probably, but the spec is useful also to navigate function usages through the whole project and keep all clear at first sight so I like to keep it

Reply:
Makes sense!